### ISSUE TITLE
Same values for `mean` and `sum` reductions of TV loss. 

### ISSUE THREAD

**Issue by denproc** [photosynthesis-team/photosynthesis.metrics#65]
**Describe the bug**
Reductions at `TVLoss` return the same value, when default parameters are used to initialise `TVLoss()`

**To Reproduce**
```python
from photosynthesis_metrics import TVLoss
import torch
torch.manual_seed(42)
a = torch.rand(3,3,256,256)
print(f'mean reduction: {TVLoss(reduction='mean')(a)}')
print(f'sum reduction: {TVLoss(reduction='sum')(a)}')
```
Output
```python
>>> mean reduction: 255.3401
>>> sum reduction: 255.3401
```

**Expected behavior**
Output
```python
>>> mean reduction: 255.3401
>>> sum reduction: 766.0204
```

**Solution**
- Get rid of `size_average` parameter at `TVLoss`;
- Call `total_variation` with `size_average=False` at `TVLoss().forward()`
